autogen/agentchat/contrib/agent_builder.py

```diff
@@ -762,3 +824,41 @@ def load(
                 }
             )
             return self._build_agents(use_oai_assistant, **kwargs)
+
+# Test
```

Comment:
could you move this test to `https://github.com/microsoft/autogen/blob/main/test/agentchat/contrib/test_agent_builder.py`?